src/IO/H5/Python/ExtendConnectivityData.py

```diff
@@ -0,0 +1,56 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+
+import click
+import rich
+
+import spectre.IO.H5 as spectre_h5
+
+
+@click.command()
+@click.argument(
+    "filename",
```

Comment:
Hi, could we discuss this once? After speaking to Larry and Nils Deppe about this, we realised it would be better if the extend connectivity runs on one file only, but the combine_h5 CLI (once its made) will run on multiple files, after which the extend connectivity will be applied to the combined h5.